hoot-core/src/main/cpp/hoot/core/ops/SmallWayMerger.cpp

```diff
@@ -91,8 +91,9 @@ void SmallWayMerger::apply(boost::shared_ptr<OsmMap>& map)
     {
       boost::shared_ptr<Way> w = it->second;
 
-      // if the way is smaller than the threshold
+      // if the way is smaller than the threshold, that isn't a `hoot:special` way
       if (highwayCrit.isSatisfied(w) &&
+          !w->getTags().contains(MetadataTags::HootSpecial()) &&
```

Comment:
I found that sometimes small roundabout connector ways were being merged by the `SmallWayMerger`, that isn't a good idea.  They are cleaned up later when the roundabouts are cleaned up.